src/main/app/case/answers/getAnswerRows.ts

```diff
@@ -27,17 +28,43 @@ export const getAnswerRows = function (section: Sections): GovUkNunjucksSummary[
     .filter(step => step.showInSection === section)
     .flatMap(step => {
       const fieldKeys = Object.keys(step.form.fields);
-      const stepContent = {
-        ...this.ctx,
-        ...generatePageContent({
-          language,
-          pageContent: step.generateContent,
-          isDivorce,
-          formState: processedFormState,
-          userEmail,
-        }),
-      };
+      let stepContent;
+      try {
+        stepContent = {
+          ...this.ctx,
+          ...generatePageContent({
+            language,
+            pageContent: step.generateContent,
+            isDivorce,
+            formState: processedFormState,
+            userEmail,
+          }),
+        };
+      } catch {
+        // Some steps may throw an error if the user is not allowed to view them yet
```

Comment:
didnt know we had pages like this as so far, i just enter the url and it works, even in the functional test, we just enter the url.

Reply:
The "You can use courts in England/Wales" page checks if they have a valid connection to England/Wales before being displayed otherwise the dynamic content will generate an empty page (if they've not completed the jurisdiction section).